patches/chromium/fix_media_key_usage_with_globalshortcuts.patch

```diff
@@ -56,7 +56,7 @@ index f96af14cb915c8ab92b314ac15b6dffcdd6ec607..2556abe433493251ebd48d4aeddaa6d4
 +  static void SetShouldUseInternalMediaKeyHandling(bool should_use);
 +
    // Register an observer for when a certain |accelerator| is struck. Returns
-   // true if register successfully, or false if 1) the specificied |accelerator|
+   // true if register successfully, or false if 1) the specified |accelerator|
```

Comment:
This typo belongs to Chromium's file `chrome/browser/extensions/global_shortcut_listener.h` that is being patched here. We need to leave the spelling alone in order for the patch to succeed